crrExistingObjects.js

```diff
@@ -13,6 +13,7 @@ const ACCESS_KEY = process.env.ACCESS_KEY;
 const SECRET_KEY = process.env.SECRET_KEY;
 const ENDPOINT = process.env.ENDPOINT;
 const SITE_NAME = process.env.SITE_NAME;
+let STORAGE_TYPE = process.env.STORAGE_TYPE;
```

Comment:
Could this be a CLI option the same way as `BUCKETS`?

Reply:
I was modeling it off of `TARGET_REPLICATION_STATUS` because it is also related to metadata fields in `replicationInfo`. I could be convinced to use a positional argument though 😄. (I was also considering of moving over to a more user-friendly interface using `commander`, granted there's enough time.)

Reply:
I mean I think ideally the `TARGET_REPLICATION_STATUS` would have a positional argument as well but since it's a fairly safe default to work out of the box I think it doesn't matter nearly as much. Either way I think it'll be fine, just looking at it from the user-friendliness side.